app/views/admin/shelters/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>All Shelters</h1>
+
+<%= link_to "Sort by number of pets", "/shelters?sort=pet_count" %>
+
+<%= form_with url: "/shelters", method: :get, local: true do |f| %>
+  <%= f.label :search %>
+  <%= f.text_field :search %>
+  <%= f.submit "Search" %>
+<% end %>
+
+<% @shelters.each do |shelter| %>
+  <div id="shelter-<%= shelter.id %>">
+    <h3><%= shelter.name %></h3>
+    <p>Created at: <%= shelter.created_at %></p>
+    <%= link_to "Update #{shelter.name}", "/shelters/#{shelter.id}/edit" %>
```

Comment:
These are just little add-ins from you right? Or is it necessary for this story/others? Think it's awesome if you're adding little usability bits in just for the sake of it!